MoonDawgArduino/MoonDawgArduino/MoonDawgArduino.ino

```diff
@@ -29,13 +36,24 @@ constexpr char VERTICAL = 'e';
 constexpr char ARM = 'a';
 
 
-// Class constructors
+/* 
+* Class constructors
+* Default constructors are only for "creating" the class 
```

Comment:
what does this comment contribute

Reply:
It's something we didn't know when we were making the classes and how we needed a default constructor for it to work

Reply:
Explain that instead. E.g. "Default constructors are required here without arguments. Initializing the motor functions will cause something to break"

Reply:
Agreed, done